build_runner/lib/src/package_graph/package_graph.dart

```diff
@@ -191,20 +191,20 @@ class PackageNode {
   ///
   /// Only the files matching these patterns will end up in the asset graph,
   /// unless they also match a pattern in [excludes].
-  final List<String> _includes;
-  Iterable<String> get includes => _includes;
+  @deprecated
+  final Iterable<String> includes;
 
   /// The glob patterns to exclude from this package.
   ///
   /// Any files matching these pattern will be excluded from the asset graph.
-  final List<String> _excludes;
-  Iterable<String> get excludes => _excludes;
+  @deprecated
+  final Iterable<String> excludes;
 
   PackageNode(this.name, this.version, this.dependencyType, String path,
       {Iterable<String> includes, Iterable<String> excludes})
```

Comment:
These should be deprecated as well now right? Does passing them have any effect?

Reply:
Only insomuch as if some other code is reading these they are still tracked.

Reply:
My point though is that this is actually a breaking change - since it no longer has an affect on the build and it used to.

Reply:
Ah, good point. Maybe we need to hold this until we start pushing through the breaking changes.